<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options_Menu</class>
 <widget class="QWidget" name="Options_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Options Menu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Appearance</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>201</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Piece Graphics</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QRadioButton" name="PG_ChessStyle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>Chess Style</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="PG_ShogiStyle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>Shogi Style</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>221</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Coordinates</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string>Horizontal</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QRadioButton" name="Coords_H19">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>61</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>1 - 9</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="Coords_Hai">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>51</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>a - i</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>111</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string>Vertical</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QRadioButton" name="Coords_Vik">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>一  -   九</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="Coords_V19">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>61</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>1 - 9</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="board_preview">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../resources/external-content.duckduckgo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPresets">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Gameplay Presets...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReturn">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
